wger/core/management/commands/list-user-rest-api.py

```diff
@@ -0,0 +1,31 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.contrib.auth.models import User
+from wger.core.models import UserModel
+from tabulate import tabulate
+
+
+class Command(BaseCommand):
+
+    help = 'Get all users created via the API'
+
+    def add_arguments(self, parser):
+        parser.add_argument('createdby_username', type=str)
```

Comment:
@Emmanuel-Dominic the argument createdby_username doesn't match the one in the description of your PR

Reply:
This `createdby_username` is just an argument field, it works similarly like a `variable` rather not the actual value or field for any Model.

Reply:
In your PR description, I see commands like `python3 manage.py add-user-rest-api api_key username_for_the_creator` meaning variable `api_key` is and `username_for_the_creator` are passed to this command. How is `username_for_the_creator` the same as `createdby_username`?

Reply:
I have changed that to `creator_username`, thanks